src/dialect/mod.rs

```diff
@@ -601,13 +601,122 @@ pub trait Dialect: Debug + Any {
         false
     }
 
-    /// Return true if the dialect supports specifying multiple options
+    /// Returns true if the dialect supports specifying multiple options
     /// in a `CREATE TABLE` statement for the structure of the new table. For example:
     /// `CREATE TABLE t (a INT, b INT) AS SELECT 1 AS b, 2 AS a`
     fn supports_create_table_multi_schema_info_sources(&self) -> bool {
         false
     }
 
+    /// Returns `true` if the dialect supports qualified column names
```

Comment:
I think if the qualified identifers in this context doesn't introduce any ambiguous grammar in other dialects we can have the parser always accept it and avoid this dialect method

Reply:
Yeah as long as the syntax doesn't collide with any reserved feature (usually the parsing logic will highlight if that's the case or the test will fail when run against that dialect) it would be fine to always accept a qualified identifier regardless of dialect.